app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BookingsController < ApplicationController
+  def create
+    @booking = Booking.new
+    @user = current_user
+    @lesson = Lesson.find(params[:lesson_id])
+    @booking.lesson = @lesson
+    @booking.user = @user
+    @booking.save!
+    if current_user
+      @booking.save!
+      redirect_to user_path(@user)
+    else
+      redirect_to new_user_session_path
```

Comment:
OK for now, possibly improve by providing a way to return to lesson page after login.